packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -379,27 +440,27 @@ class IrisGridTableModel extends IrisGridModel {
         value,
         column.type,
         column.name,
-        formatOverride
+        formatOverride ?? undefined
       );
       this.cacheFormattedValue(x, y, text);
     }
 
     return this.formattedStringData[x][y];
   }
 
-  textForCell(x, y) {
+  textForCell(x: ModelIndex, y: ModelIndex): string | null {
```

Comment:
Shouldn't allow `null` to be returned here, add `return text ?? ''` at the bottom.